extension/src/popup/components/SelectionTile/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Button, Icon } from "@stellar/design-system";
+
+import "./styles.scss";
+
+interface SelectionTileProps {
+  icon: React.ReactNode;
+  primaryText: string;
+  secondaryText?: string;
+  onClick: () => void;
+  isEmpty?: boolean;
+  useIconWrapper?: boolean;
+  testId?: string;
+}
+
+export const SelectionTile = ({
+  icon,
+  primaryText,
+  secondaryText,
+  onClick,
+  isEmpty = false,
+  useIconWrapper = true,
```

Comment:
NIT: can we call this `shouldUseIconWrapper` or something similar since the `use-` prefix is reserved for React hooks